### DISCUSSION TITLE
Point Cloud Data

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1067] [Q&A] [answered]
i change the osgb to 3dtiles,and load success by attachRealityModel method
now,i change the Point Cloud Data(las format) to 3dtiles, but not load success  in imodeljs,
who can help me ?

**Comment by pmconne** [comment]
You need to provide more information.

**Reply by paprophet** [reply]
the web netwok show load tileset.json and 0.json ,but not load other json data or pnts data  and not show Point Cloud, 
give me emai address l, i can send data to you ! thank you !

**Reply by pmconne** [reply]
Can you upload it somewhere please?

**Reply by paprophet** [reply]
It's very slow to visit the github,so  can't  submit the data to there ,you can download this 
linkurl：https://pan.baidu.com/s/1xFYbuCDINoBCArWy0yf2Wg 
key：dnzj

**Reply by pmconne** [reply]
It wants me to install Baidu Netdisk to download the folder. I don't want to install that. If I try to download the zip file, it prompts me for an account name and password, which I don't have.

Have you checked the browser console for errors? Have you tried using the debugger? Are you able to display the 3d tiles in a different viewer, like Cesium? How did you convert the LAS to 3d tiles?

**Reply by paprophet** [reply]
get me a mail,i want send the point cloud data to you, 
in cesium is success,
use the tools from http://www.cesiumlab.com/
and   the osgb change to 3d tiles success  use the tools too.

![image](https://user-images.githubusercontent.com/18022720/112995356-b5beb780-919d-11eb-9eaa-5a4e078f19b4.png)

**Reply by pmconne** [reply]
Send it to pc4868696@gmail.com

**Reply by paprophet** [reply]
i send the data to your email ,thank you!

**Comment by pmconne** [comment]
@paprophet The problem appears to be that your JSON contains URIs that are relative to the location of the JSON file within the directory structure. Looking into whether/how that's supposed to work.

EDIT: Actually it looks like it might be because the root tile's content is another JSON file. That's supposed to work.

**Reply by paprophet** [reply]
today，i update the JSON files in other version. but is not success. so  please change some JSON file (not change the  directory structure) and test. if success send the file to my email . thank you！

**Reply by RBBentley** [reply]
@paprophet  - It appears that the case where the root node points to another JSON file is not handled correctly.   I'm working on correcting this and will provide an update when this is resolved.

**Reply by paprophet** [reply]
hope to solve this problem in new version of imodeljs ,thank you !

**Reply by RBBentley** [reply]
If you would like to try this before the PR is merged, the fix branch is here. https://github.com/imodeljs/imodeljs/tree/Fix-point-cloud-issues

**Reply by RBBentley** [reply]
This has been comitted to master here. https://github.com/imodeljs/imodeljs/commit/52478d10f2ac048be64307a4644f88e93d155352 and should be in next version.

**Reply by paprophet** [reply]
now the problem exist , i update imodeljs to 2.15.1
and i send the point cloud data to your email @pmconne 
thank you!
![QQ图片20210507171108](https://user-images.githubusercontent.com/18022720/117427605-ca8a2a00-af57-11eb-971f-dbdfe56bfdf9.png)

**Reply by pmconne** [reply]
@paprophet Your image shows Autzen Stadium which to my knowledge is in Oregon, USA. When I attach your point cloud in my app and fit the view, it takes me to Beijing and shows your point cloud floating above the ground there. Where are you expecting the point cloud to be located?

NOTE: I used the point cloud data you previously supplied. If you've got a new problem with a new point cloud, start a new discussion.

**Reply by paprophet** [reply]
Your image shows Autzen Stadium which to my knowledge is in Oregon, USA. When I attach your point cloud in my app and fit the view, it takes me to Beijing and shows your point cloud floating above the ground there. Where are you expecting the point cloud to be located? ----- can you give me a pic of  this.
and i do not know how to fit the view by the point cloud data.(the point cloud data not have Latitude and longitude)

**Reply by pmconne** [reply]
![image](https://user-images.githubusercontent.com/22944042/117442724-25466600-af05-11eb-943f-7f38c0b4659a.png)

In the image above your point cloud is inside the green box.
You should be able to fit the view by double-clicking the middle mouse button. If you turn off all other models in the view other than your point cloud it should fit directly to the point cloud.

**Comment by RBBentley** [comment, marked as answer]
This has been fixed in this commit.  https://github.com/imodeljs/imodeljs/commit/52478d10f2ac048be64307a4644f88e93d155352  It should be available in next version.
